dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/EnrollmentQueryParams.java

```diff
@@ -95,6 +95,9 @@ class EnrollmentQueryParams {
   /** Indicates whether to include soft-deleted enrollments */
   private boolean includeDeleted;
 
+  /** Indicates whether to include tracked entity attribute data in the enrollment */
+  private boolean includeAttributes;
```

Comment:
Is there a reasoning behind this? Will this be a breaking change?

Reply:
The reason of this flag is because I want to get the enrollment attributes from the database only when the user requests them. 
It's not a breaking change, we currently support this on the API level if the user includes `fields=attributes` in the request. With Hibernate we didn't use it, because the attributes were loaded lazily, but we do need it with JDBC.

Reply:
I mean. What was the previous behaviour in the API? if attributes was not explicitly requested as part of fields queryParam, we never sent it? Was the default that we did not send the attributes?

Reply:
Yes, this behavior didn't change. 
Previously, we only returned attributes if explicitly requested, as in `fields=attributes`. By default we didn't return them.